src/test/core/SociDB_test.cpp

```diff
@@ -321,8 +321,8 @@ class SociDB_test final : public TestSuite
                 LedgerSeq       BIGINT UNSIGNED             \
             );",
                 "CREATE INDEX SeqLedger ON Ledgers(LedgerSeq);"};
-            int dbInitCount = std::extent<decltype(dbInit)>::value;
-            for (int i = 0; i < dbInitCount; ++i)
+            auto dbInitCount = std::size(dbInit);
+            for (auto i = 0u; i < dbInitCount; ++i)
```

Comment:
No change needed, but why not `for (auto const s : dbInit)`?

Reply:
That would be good too.  I'll go ahead and change it.  I was so intent on just doing the mechanical change at hand I didn't take my blinders off and look at the surrounding code.